src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -85,11 +85,17 @@
         {{ form_widget(attribute(form.parent, multistoreCheckboxName)) }}
       {% endif %}
 
+      {% set disablingToggleName = disabling_toggle_prefix ~ form.vars.name %}
+      {% if attribute(form.parent, disablingToggleName) is defined %}
+        {{ form_errors(attribute(form.parent, disablingToggleName)) }}
+        {{ form_widget(attribute(form.parent, disablingToggleName)) }}
+      {% endif %}
+
       {% if label_tag_name is not defined %}
         {{ form_label(form) }}
       {% endif %}
-      <div class="{{ block('form_group_class') }}{% if disabledField %} disabled{% endif %}">
-        {{ form_widget(form) }}
+      <div class="{{ block('form_group_class') }}">
```

Comment:
Be careful you must not remove this class which is needed when inputs are disabled on first form rendering (it is used in multishop configuration forms) However this could be nice to move this into the `form_group_class` block along with the loop that sets the `disabledField` variable

Reply:
I removed only div class, the inputs are still disabled. It doesn't behave very well if we disable whole divs and its hard to implement reusable disablingToggle that way).